.github/workflows/build-timestamped-master.yml

```
@@ -0,0 +1,16 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
```

```suggestion
      - main
```
let's check and change other places